src/components/autocompleter/autocompleter.jsx

```diff
@@ -0,0 +1,106 @@
+import PropTypes from 'prop-types';
+import React, { forwardRef, useEffect, useState } from 'react';
+import useFetch from '@bloodyaugust/use-fetch';
+import Select from '../select/select.jsx';
+import ListOption from '../list-option/list-option.jsx';
+import mockConstants from '../../mocks/mock-constants.js';
+
+const { API_ROOT } = mockConstants;
+
+const Autocompleter = forwardRef(function Autocompleter(props, ref) {
+  const { execute: fetch } = useFetch();
+  const [models, setModels] = useState(props.models);
+
+  useEffect(() => {
+    fetchModels();
+  }, []);
+
+  function fetchModels(searchString) {
+    fetch(apiEndpoint(searchString))
+      .then((results) => {
+        setModels(results.json.results.map(resultInfo => results.json[resultInfo.key][resultInfo.id]));
+      }).catch((error) => {
+        if (error.error && error.error.type !== 'aborted') {
```

Comment:
Do we have aborting requests set up? Should we?

I think in bigmaven it's easy to do, but I wonder if it makes sense for that to be part of the design system component

Reply:
That's a good point. We have abort requests when the component unmounts. But, AFAIK, we need a concept for aborting requests on consecutive interactions. In this case, we need to abort (and probably debounce) any requests when making a new one while the user types.

Reply:
I think for request aborting, we can do it in a new PR. That way, we keep this PR focused on a basic autocompleter and we can introduce better functionality as iterations. 

What do y'all think?

Reply:
We can totally expose a method to enable this behavior in `useFetch`. It's not there because we didn't need it yet, but if we have a need now... No time like the present!